registry/src/store/actions/searchActions.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+
+export const SEARCH_PACKAGE = "SEARCH_PACKAGE";
+export const SEARCH_SUCCESS = "SEARCH_SUCCESS";
+export const SEARCH_FAILURE = "SEARCH_FAILURE";
+
+export const searchPackage =
+  (query, page, sorted_by = "") =>
+  async (dispatch) => {
+    if (sorted_by === "Date last updated") {
+      sorted_by = "updatedat";
+    } else {
+      sorted_by = "";
+    }
+
+    const url = `${
+      process.env.REACT_APP_REGISTRY_API_URL
+    }/packages?query=${encodeURIComponent(query)}&page=${encodeURIComponent(
+      page
+    )}&sorted_by=${encodeURIComponent(sorted_by)}`;
```

Comment:
@arteevraina as we have implemented the functionality of sort with ascending and descending , we could also provide options for that in UI.